haskell-indentation.el

```diff
@@ -1,9 +1,11 @@
 ;;; haskell-indentation.el -- indentation module for Haskell Mode
 
-;; Copyright (C) 2009  Kristof Bastiaensen
+;; Copyright (C) 2013  Kristof Bastiaensen, Gergely Risko
 
 ;; Author: Kristof Bastiaensen <kristof.bastiaensen@vleeuwen.org>
-
+;; Author: Gergely Risko <gergely@risko.hu>
```

Comment:
Can you please use errge@nilcons.com with this merge?  I know that I've used gergely@risko.hu myself, but I actually prefer errge@nilcons.com for haskell projects nowadays!